ExtractorLauncher/ExtractorStarter.cs

```diff
@@ -124,6 +125,47 @@ public static class ExtractorStarter
             {
                 return "subscriptions.keep-alive-count must be greater than 0";
             }
+#pragma warning disable 0618
+            if (config.Cognite?.RawMetadata != null)
+            {
+                log.LogWarning("cognite.raw-metadata is deprecated. Use cognite.metadata-targets instead");
+                if (config.Cognite.MetadataTargets != null)
+                {
+                    return "cognite.raw-metadata and cognite.metadata-targets cannot be set at the same time.";
+                }
+                if (config.Cognite == null) config.Cognite = new CognitePusherConfig();
+                var rawMetadata = config.Cognite.RawMetadata;
+                var useCleanAssets = (rawMetadata?.Database == null || rawMetadata?.AssetsTable == null) || config.Cognite.SkipMetadata;
+                var useCleanTimeseries = rawMetadata?.Database == null || rawMetadata?.TimeseriesTable == null;
+                var useCleanRelationships = rawMetadata?.Database == null || rawMetadata?.RelationshipsTable == null;
+                config.Cognite.MetadataTargets = new MetadataTargetsConfig
+                {
+                    CleanMetadata = new CleanMetadataTargetConfig
+                    {
+                        Assets = useCleanAssets,
+                        Timeseries = useCleanTimeseries,
+                        Relationships = useCleanRelationships
+                    },
+                    RawMetadata = new RawMetadataTargetConfig
+                    {
+                        Database = rawMetadata?.Database,
+                        AssetsTable = rawMetadata?.AssetsTable,
+                        TimeseriesTable = rawMetadata?.TimeseriesTable,
+                        RelationshipsTable = rawMetadata?.RelationshipsTable
+                    }
+                };
+            }
```

Comment:
```c#
else if (config.Cognite.MetadataTargets == null) {
    if (skipMetadata) {
        Deprecation warning
    }
    else {
         Enable clean by default + deprecation warning Default writing to clean is deprecated, in the future not setting a metadata target will not write metadata to CDF at all
    }
}
```

Reply:
Also if this is the case log

`Default writing to clean is deprecated, in the future not setting a metadata target will not write metadata to CDF at all`